src/main/java/core/basesyntax/storage/ListStorageBets.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.storage;
+
+import core.basesyntax.model.Bet;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListStorageBets {
```

Comment:
Suggestion: change ListStorageBets to BetStorage

Reply:
It is plane list it is better to see what it is, storage could be db for example.